SECURITY.md

```diff
@@ -17,9 +17,17 @@ This is a shared security policy for the CosmWasm stack, including the following
 
 ## Reporting a Vulnerability
 
-Please report any security issues via email to security@confio.gmbh.
+There are three ways to report a security issue
 
-You will receive a response from us within 4 working days confirming that a human read your email. If you do not hear back within 1 week, feel free to send a reminder or try to notify core team members via different channels.
+|                     | Cosmos HackerOne Bug Bounty program                   | security<span>@</span>interchain.io                                                                                                                                                                                                                                                                 | security<span>@</span>confio.gmbh                                                                                                                                                                                                 |
+| ------------------- | ----------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Maintained by       | Amulet                                                | Amulet                                                                                                                                                                                                                                                                                              | Confio                                                                                                                                                                                                                            |
+| Eligable for bounty | yes                                                   | no                                                                                                                                                                                                                                                                                                  | no                                                                                                                                                                                                                                |
```

Comment:
```suggestion
| Eligible for bounty | yes                                                   | no                                                                                                                                                                                                                                                                                                  | no                                                                                                                                                                                                                                |
```

Reply:
Thanks – I always mess up this word

Reply:
Now let's see the autofixer in action for re-formatting the table ...

Reply:
Nevermind, no change needed